src/organisms/forms/AddForm/index.js

```diff
@@ -60,7 +62,7 @@ AddForm.propTypes = {
   /**
    * determines if the close icon is present
    */
-  closeIcon: PropTypes.bool.isRequired,
+  closeIcon: PropTypes.bool,
```

Comment:
`.isRequired`? (also if the answer is the same for all of these you don't have to comment, just want to make sure none of them slip through the cracks

Reply:
see above